src/Resources/config/pimcore/config.yml

```
@@ -1,3 +1,15 @@
+framework:
+    messenger:
+        transports:
+            pimcore_index_queues:
+                dsn: 'doctrine://default'
+                options:
+                    queue_name: pimcore_index_queues
```

This would allow overriding the default transport. please see https://github.com/pimcore/pimcore/blob/10.x/bundles/CoreBundle/Resources/config/pimcore/default.yaml#L33
```suggestion
            pimcore_index_queues: 'doctrine://default?queue_name=pimcore_index_queues'
```

---

> Should we also add `symfony/messenger` requirement here? just thinking about how it will work on version 6.9

added composer dependency. 

overwriting transport config still possible... e.g. like

```yml
pimcore_index_queues:
   dsn: 'redis-bla-something'
   options: ~
```